File: abc/fukushu/169d（だめ）.py
def prime_numbers(m):
    # 素数を保持するset
    primes = set()

    # 2以上m以下の素数を数え上げる
    n = 2
    while n <= m:
        # 自身より小さい全ての素数で割り切れなければ、素数とみなせる
        if all(n % i != 0 for i in primes):
            primes.add(n)
        n += 1

    return primes

File: abc/fukushu/169d（だめ）.cpp
#include <bits/stdc++.h>
using namespace std;
#define rep(i,n) for (int i=0; i < (int)(n); i++)
#define fo(i,s,e) for (int i=(s); i < (e); i++)
#define all(obj) (obj).begin(), (obj).end()
#define int long long
signed main () {
  int n;cin>>n;
  int ans=0;
  int k=1;
  if (n==1){cout<<0;return 0;}
  while (n!=1) {
    fo(i,k+1,n+1){
      if(n%i==0){
        ans+=1;
        n=n/i;
        k=i;
        break;
      }
      else if (k==n) {
        return 0;
      }
    }
  }
  cout<<ans;
}